bofire/strategies/stepwise/stepwise.py

```diff
@@ -41,40 +43,67 @@
 }
 
 
-def map(data_model: data_models.Strategy) -> Strategy:
+def _map(data_model: data_models.Strategy) -> Strategy:
     cls = STRATEGY_MAP[data_model.__class__]
     return cls.from_spec(data_model=data_model)
 
 
+T = TypeVar("T", pd.DataFrame, Domain)
+
+
+TfData = Union[Literal["experiments"], Literal["candidates"], Literal["domain"]]
+
+
+def _apply_tf(
+    data: Optional[T],
+    transform: Optional[Transform],
+    tf: TfData,
+) -> Optional[T]:
+    if data is not None and transform is not None:
+        return getattr(transform, f"transform_{tf}")(data)
+
+
 class StepwiseStrategy(Strategy):
     def __init__(self, data_model: data_model, **kwargs):
         super().__init__(data_model, **kwargs)
-        self.steps = data_model.steps
+        self.stratgies = [_map(s.strategy_data) for s in data_model.steps]
+        self.conditions = [s.condition for s in data_model.steps]
+        self.transforms = [
+            s.transform and transforms.map(s.transform) for s in data_model.steps
```

Comment:
What is the `and` doing? I do not get it :D

Reply:
> What is the `and` doing? I do not get it :D

If `s.transform` is `None` we keep `None`. If it is not `None` the last value is evaluated and returned.
```
assert (None and 1) is None
assert (None or 1) == 1
```
https://en.wikipedia.org/wiki/Short-circuit_evaluation